src/networking.c

```diff
@@ -565,7 +658,17 @@ cc_oci_network_discover(struct cc_oci_config *const config,
 			ipv4_cfg->subnet_mask = cc_net_get_ip_address(family,
 				&((struct sockaddr_in *)ifa->ifa_netmask)->sin_addr);
 
-
+			/* Retrieve the mtu set by docker for the interface to be set
```

Comment:
Just a non-blocking nitpick - docker is not the only thing that might be calling us ;-) We can now run under CRI-O and k8s for instance...